Documentation/CommandReference/Index.rst

```diff
@@ -0,0 +1,736 @@
+
+.. include:: ../Includes.txt
+
+
+.. _typo3_console-command-reference:
+
+Command Reference
+=================
+
+.. note:
+
+  This reference uses ``./typo3cms`` as the command to invoke. If you are on
+  Windows, this will probably not work, there you need to use ``typo3cms.bat``
+  instead.
+
+
+The following reference was automatically generated from code on 21-06-16
```

Comment:
Can we use ISO date here, thus `2016-06-21`? And maybe add the current time too.